scipy/spatial/_plotutils.py

```diff
@@ -153,6 +153,28 @@ def voronoi_plot_2d(vor, ax=None, **kw):
     -----
     Requires Matplotlib.
 
+    Examples
+    --------
+    Set of point:
+
+    >>> import matplotlib.pyplot as plt
+    >>> points = np.random.rand(10,2) #random
+
+    Voronoi diagram of the points:
+
+    >>> from scipy.spatial import Voronoi, voronoi_plot_2d
+    >>> vor = Voronoi(points)
+
+    using `voronoi_plot_2d` for visualisation:
+
+    >>> voronoi_plot_2d(vor)
```

Comment:
you still need to do `fig = voronoi_...` here. And same 4 lines lower. that will fix the CI failures.